court/tests.py

```diff
@@ -127,3 +127,7 @@ def test_court_without_max_players(self):
                 city='TEST_CITY',
                 neighborhood='TEST_NEIGHBORHOOD'
             )
+
+    def test_get_all_courts(self, court):
+        courts = Court.get_all_courts()
+        assert court in courts
```

Comment:
You're testing that court is there rather than all courts. To test it in a robust way, you'd need to use the actual line that you use inside the method. That raises the question, do you really gain anything by adding this method? You could just replace any call to this custom method to a call to the built-in method that this custom method is calling. No? It will be just as easy to use and will mean less code to maintain.

Reply:
True, will be replaced